2024-02-19 20:35:25,026 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-19 20:35:25,026 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Configure stats pid to 19448
2024-02-19 20:35:25,026 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-19 20:35:25,026 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-19 20:35:25,027 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-19 20:35:25,027 INFO    MainThread:19448 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-19 20:35:25,027 INFO    MainThread:19448 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_203525-cuvzg9mo\logs\debug.log
2024-02-19 20:35:25,027 INFO    MainThread:19448 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240219_203525-cuvzg9mo\logs\debug-internal.log
2024-02-19 20:35:25,029 INFO    MainThread:19448 [wandb_init.py:init():566] calling init triggers
2024-02-19 20:35:25,029 INFO    MainThread:19448 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_samples': 2, 'max_depth': 1}
2024-02-19 20:35:25,029 INFO    MainThread:19448 [wandb_init.py:init():616] starting backend
2024-02-19 20:35:25,029 INFO    MainThread:19448 [wandb_init.py:init():620] setting up manager
2024-02-19 20:35:25,031 INFO    MainThread:19448 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-19 20:35:25,031 INFO    MainThread:19448 [wandb_init.py:init():628] backend started and connected
2024-02-19 20:35:25,039 INFO    MainThread:19448 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-19 20:35:25,039 INFO    MainThread:19448 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-19 20:35:25,040 INFO    MainThread:19448 [wandb_init.py:init():720] updated telemetry
2024-02-19 20:35:25,041 INFO    MainThread:19448 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-19 20:36:55,958 ERROR   MainThread:19448 [wandb_init.py:init():779] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-02-19 20:36:56,281 ERROR   MainThread:19448 [wandb_init.py:init():1194] Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "f:\test\lib\site-packages\wandb\sdk\wandb_init.py", line 1176, in init
    run = wi.init()
  File "f:\test\lib\site-packages\wandb\sdk\wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
